src/it/java/org/owasp/webgoat/ChallengeIT.java

```diff
@@ -0,0 +1,112 @@
+package org.owasp.webgoat;
+
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class ChallengeIT extends IntegrationTest {
```

Comment:
In my opinion, I would like not to use abbreviations and follow the Java Unit Naming Convention for test case classes.
Like ChallengeIntegrationTest.
The comment applies for all the rest of the test classes.